main.go

```diff
@@ -18,6 +18,9 @@ import (
 func main() {
 	config := common.GetConfig()
 	logging.SetLevel(*config.LogLevel)
+	if err := logging.SetOutput(*config.LogOutput); err != nil {
```

Comment:
This will panic if the config doesn't include a `logOutput` element. You could add a nil check (or check if it's an empty string if you change the type like I suggested) at the end of the common.GetConfig function

Reply:
Addressed in commit [57cfb77](https://github.com/WiiLink24/wfc-server/pull/48/commits/57cfb77af059763da33748a6f72c4d5cf4cc9666).